### DISCUSSION TITLE
Tuya outlet energy monitoring

### DISCUSSION THREAD

**Discussion by Jonas-2811** [milo526/homebridge-tuya-web#503] [Q&A]
Hey,
is there a possibility to see the energy output on tuya outlets with your tuyaweb-plugin?

**Comment by milo526** [comment]
As far as I know Tuya does not expose that information through the API that this plugin uses.

Please post the result of [the steps under "How to check whether the api this library uses can control your device"](https://github.com/milo526/homebridge-tuya-web#how-to-check-whether-the-api-this-library-uses-can-control-your-device) to double check my assumption.
